Source/WebCore/Modules/mediastream/RTCStatsReport.h

```diff
@@ -32,6 +32,14 @@
 #include "RTCIceTcpCandidateType.h"
 #include "RTCIceTransportState.h"
 #include <wtf/KeyValuePair.h>
+#include <wtf/RefCounted.h>
+#include <wtf/text/WTFString.h>
+
+#if USE(GSTREAMER_WEBRTC)
+#define GST_USE_UNSTABLE_API
+#include <gst/webrtc/webrtc.h>
+#undef GST_USE_UNSTABLE_API
+#endif
```

Comment:
Need the `#include` here instead of forward declaring types because of using enum type `GstWebRTCStatsType`.